system/assets/css/admin/specific/rolepicker/rolepicker.less

```diff
@@ -0,0 +1,22 @@
+@import url( 'globals.less' );
```

Comment:
Given this is specific css for a role picker, it would make sense to wrap the whole thing in a containing div with a `preside-role-picker` class so that the css remains specific? So add that class to the css + add a wrapping div to the markup.